src/hotspot/share/opto/macro.hpp

```diff
@@ -196,7 +193,7 @@ class PhaseMacroExpand : public Phase {
                           Node* klass_node, Node* length,
                           Node* size_in_bytes);
 
-  Node* make_arraycopy_load(ArrayCopyNode* ac, intptr_t offset, Node* ctl, Node* mem, BasicType ft, const Type *ftype, AllocateNode *alloc);
+  static Node* make_arraycopy_load(Compile* comp, PhaseIterGVN* igvn, ArrayCopyNode* ac, intptr_t offset, Node* ctl, Node* mem, BasicType ft, const Type *ftype, AllocateNode *alloc);
```

Comment:
Is this just cleanup? The method should have always been static?

Reply:
This method was made static because it's used by other static methods.